petab/core.py

```diff
@@ -90,6 +101,94 @@ def get_notnull_columns(df: pd.DataFrame, candidates: Iterable):
             if col in df and not np.all(df[col].isnull())]
 
 
+def get_replacement_id(groupvars, groupvar) -> str:
```

Comment:
```suggestion
def get_observable_replacement_id(groupvars, groupvar) -> str:
```
More coherent with the following function?